javascript/ql/src/Summary/LinesOfCode.ql

```diff
@@ -0,0 +1,11 @@
+/**
+ * @id js/summary/lines-of-code
+ * @name Total lines of JavaScript and TypeScript code in the database
```

Comment:
Nit: Should this be hedged to account for the "technically-not-JavaScript" code from Angular and similar related languages that we handle? (Given the debugging purpose of this query, I think it is fine to count a few more lines than what the name promises.)

Reply:
I'm more worried about us overthinking this and coming up with a confusing name that nobody understands. I'd rather keep it like this.

Reply:
Ok. You have the approval already.